### PR TITLE
vp9hdec: clean compile warnings

### PR DESCRIPTION
**Pull request by lsun30**
Signed-off-by: Lina Sun lina.sun@intel.com

### CODE REVIEW

File: src/vp9hdec/decode_hybrid_vp9.cpp [hunk 1]
```diff
@@ -1968,7 +1880,7 @@ VAStatus Intel_HybridVp9Decode_MdfHost_Create (
         driver_context.shared_bufmgr = 1;
 
 	cm_version = CM_4_0;
-        cm_status = CreateCmDevice(pMdfDevice, cm_version, &driver_context, CM_DEVICE_CREATE_OPTION_FOR_VP9);
+        CreateCmDevice(pMdfDevice, cm_version, &driver_context, CM_DEVICE_CREATE_OPTION_FOR_VP9);
```
Comment (wlin70) [inline src/vp9hdec/decode_hybrid_vp9.cpp]: Please keep this error checking on CreateCmDevice

Comment (lsun30) [inline src/vp9hdec/decode_hybrid_vp9.cpp]: I have updated the code with error checking of CreateCmDevice, please check.

**Comment by seanvk** [comment]
lgtm +2

**Comment by seanvk** [comment]
verified

**Pull request merged**